README.md

```diff
@@ -1,130 +1,182 @@
 5. Your application's backend server will need to accept the authorization code and exchange it for an access token.
 
-The SDK will help facilitate the OAuth link generation, popup dialog creation, and Smartcar will handle the user authentication and authorization. This SDK will not assist with the backend server code to accept authorization codes or exchanging for access tokens (step 6).
+## Next Steps
+
+This SDK will help facilitate OAuth link generation, popup dialog creation, and Smartcar will handle user authentication and authorization. This SDK will not assist in exchanging authorization codes for an access token. Should you choose to self host the redirect this SDK will not assist with extracting the authorization code query parameter from the redirect URI.
+
+See the links below to various SDKs for help in exchanging the authorization code for an access token. If the backend language you want to use isn't listed below you can still use Smartcar you'll just have to make HTTP requests manually based on our [API specification](https://smartcar.com/docs#get-all-vehicles).
+
+- [Node](https://github.com/smartcar/node-sdk)
+- [Python](https://github.com/smartcar/python-sdk)
+- [Java](https://github.com/smartcar/java-sdk)
+
+### Usage
+
+### Import `sdk.js`
+
+Import `sdk.js` on your client. Call the `Smartcar` constructor then use one of its instance methods to setup the authorization flow.
 
-### Import `sdk.js`:
 ```html
-<script src="https://cdn.smartcar.com/javascript-sdk/sdk-1.0.0.js"></script>
+<script src="https://javascript-sdk.smartcar.com/sdk-2.0.0.js"></script>
 ```
 
-### Use `sdk.js`:
+### Use `sdk.js`
+
 Initialize:
+
 ```javascript
 const smartcar = new Smartcar({
   clientId: `your-client-id`,
   redirectUri: `your-redirect-uri`,
-  scope: ['read_vehicle_info', 'read_odometer'], // optional
-  onComplete: function() { // optional
-    window.location.reload();
+  // optional, defaults to all scopes
+  scope: ['read_vehicle_info', 'read_odometer'],
+  // if using smartcar hosting this is required and must take at least two
+  // arguments - error & code
+  // otherwise optional (will be called with the same arguments but is not
+  // required to handle them)
+  onComplete: function(error, code, state <optional>) {
+    // actions to take on completion of auth flow
+    // if using smartcar hosting this should send the code to your backend
+    // server to exchange for an access token
```

Comment:
Do we want an example (in the previous section perhaps) how someone would do this?